tests/test_objective_funs.py

```diff
@@ -429,6 +433,57 @@ def test(eq):
         test(Equilibrium(iota=PowerSeriesProfile(0)))
         test(Equilibrium(current=PowerSeriesProfile(0)))
 
+    @pytest.mark.unit
+    def test_boundary_error_biestsc(self):
+        """Test calculation of boundary error using BIEST w/ sheet current."""
+        coil = FourierXYZCoil(5e5)
+        coilset = CoilSet.linspaced_angular(coil, n=100)
+        coil_grid = LinearGrid(N=20)
+        eq = Equilibrium(L=3, M=3, N=3, Psi=np.pi)
+        sheet_current = FourierCurrentPotentialField()
+        sheet_current.change_resolution(M=eq.M, N=eq.N, NFP=eq.NFP, sym=eq.sym)
+        eq.solve()
+        obj = BoundaryError(eq, coilset, sheet_current, field_grid=coil_grid)
+        obj.build()
+        f = obj.compute_scaled_error(*obj.xs(eq, sheet_current))
+        n = len(f) // 3
+        # first n should be B*n errors
+        np.testing.assert_allclose(f[:n], 0, atol=1e-4)
```

Comment:
can you explain this test a bit? we expect a low Bn bc it is axisymmetric in a dense solenoid? and we expect 0 K bc it is 0 pressure at edge?

Reply:
Yeah its basically just a 1/R toroidal field, so it should be tangent to the surface, and its scaled such that the pressure matches so all the errors should be small.